src/config.rs

```diff
@@ -78,6 +82,10 @@ impl ConfigBuilder {
         self
     }
 
+    pub fn set_adhoc_routing(mut self, adhoc_routing: bool) {
```

Comment:
I think the most common coding style in rust is to have this without the "set", so this would be named `adhoc_routing`

Reply:
Also, this is a Builder. This means the method must return `Self`, so that it can be chained.

Reply:
You're right. I did that in #30